### PR TITLE
OKTA-557212 Fixit Days - Add aliases for hoc references

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Fixit Days! Adding 3 aliases, one comment to a hoc reference, and one typo/broken link update
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-557212](https://oktainc.atlassian.net/browse/OKTA-557212)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ To leverage the power of [Okta Identity Engine](/docs/guides/oie-intro/) and to
 ## Plan your upgrade
 
 * **IMPORTANT**: Talk to your Okta account team and get their support and input. Okta assistance is currently a requirement for carrying out the [Okta org](/docs/concepts/okta-organizations/) upgrade process.
-* Review the [Identity Engine new features](https://help.okta.com/oie/en-us/Content/Topics/identity-engine-upgrade/features.htm) guide to get an idea of what features you are using, if any, that require remediations (meaning they need adjustment before we can proceed with an upgrade). For example, some Classic Engine features may not be supported in Identity Engine in their current form.
+* Review the [Identity Engine new features](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-features) guide to get an idea of what features you are using, if any, that require remediations (meaning they need adjustment before we can proceed with an upgrade). For example, some Classic Engine features may not be supported in Identity Engine in their current form.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:14]:
```suggestion
* Review the [Identity Engine](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-features) guide to get an idea of what features you are using, if any, that require remediations (meaning they need adjustment before we can proceed with an upgrade). For example, some Classic Engine features may not be supported in Identity Engine in their current form.
```
This HOC location is not longer about new features.

File: packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ The following table details the configurations that define which Authentication
...
+| **Redirect:<br>Okta Hosted Sign-In Widget**<br>To verify that the redirect deployment is used in your application code,<br>check that you are using one of these SDKs:<br>[Android](https://github.com/okta/okta-mobile-kotlin), [Angular](https://github.com/okta/okta-angular), [ASP.NET](https://github.com/okta/okta-aspnet), [iOS](https://github.com/okta/okta-mobile-swift), [React](https://github.com/okta/okta-react), [React Native](https://github.com/okta/okta-react-native), [Vue](https://github.com/okta/okta-vue)| The API type depends on your [org configuration.](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) | The API type depends on your [org configuration.](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md:166]:
```suggestion
| **Redirect:<br>Okta Hosted Sign-In Widget**<br>To verify that the redirect deployment is used in your application code,<br>check that you are using one of these SDKs:<br>[Android](https://github.com/okta/okta-mobile-kotlin), [Angular](https://github.com/okta/okta-angular), [ASP.NET](https://github.com/okta/okta-aspnet), [iOS](https://github.com/okta/okta-mobile-swift), [React](https://github.com/okta/okta-react), [React Native](https://github.com/okta/okta-react-native), [Vue](https://github.com/okta/okta-vue)| The API type depends on your [org configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version). | The API type depends on your [org configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version). |
```

**Review by vanngo-okta — APPROVED** [review]
LGTM! Just a couple of style nits.
Thanks for looking into these!

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md (hunk 1 above)
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:14]: This redirects to the help docs landing page. Is that what's wanted?

**Review by brentschaus-okta — APPROVED** [review]
LGTM! Just one small question.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md:14]: No. That's default fallback when alias is not working. I had to get this one created and it probably hadn't been built yet. It's loading now: https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-features

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-overview/main/index.md]: @vanngo-okta I think we still need more than "Identity Engine". I updated the link text to: "Identity Engine feature changes" to match hoc title.

File: packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md]: Fixed locally. Thanks.

File: packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md [hunk 3]
```diff
@@ -163,7 +163,7 @@ The following table details the configurations that define which Authentication
...
+| **Redirect:<br>Okta Hosted Sign-In Widget**<br>To verify that the redirect deployment is used in your application code,<br>check that you are using one of these SDKs:<br>[Android](https://github.com/okta/okta-mobile-kotlin), [Angular](https://github.com/okta/okta-angular), [ASP.NET](https://github.com/okta/okta-aspnet), [iOS](https://github.com/okta/okta-mobile-swift), [React](https://github.com/okta/okta-react), [React Native](https://github.com/okta/okta-react-native), [Vue](https://github.com/okta/okta-vue)| The API type depends on your [org configuration.](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) | The API type depends on your [org configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version). |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/redirect-vs-embedded/index.md:166]:
```suggestion
| **Redirect:<br>Okta Hosted Sign-In Widget**<br>To verify that the redirect deployment is used in your application code,<br>check that you are using one of these SDKs:<br>[Android](https://github.com/okta/okta-mobile-kotlin), [Angular](https://github.com/okta/okta-angular), [ASP.NET](https://github.com/okta/okta-aspnet), [iOS](https://github.com/okta/okta-mobile-swift), [React](https://github.com/okta/okta-react), [React Native](https://github.com/okta/okta-react-native), [Vue](https://github.com/okta/okta-vue)| The API type depends on your [org configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version). | The API type depends on your [org configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version). |
```

**Review by vanngo-okta — APPROVED** [review]
Looks good! One small nit.

**Pull request merged**